statcord/client.py

```diff
@@ -176,7 +177,26 @@ async def post_data(self) -> None:
             await self.__handle_response(resp)
 
     def start_loop(self) -> None:
-        self.bot.loop.create_task(self.__loop())
+        if self._task:
```

Comment:
Not so fast :v
If the loop is cancelled, the variable will still exist, so I need to check if the loop is actually cancelled instead of just checking the loop has been created.